### PR TITLE
Add artk053 TizenRT & romfs download mode

### PR DESCRIPTION
**Pull request by an4967**
I think we don't need to download every bin file(bl1, bl2, sssfw, wlanfw, os).
To improve development environment, adding os only download mode would be good.

### CODE REVIEW

File: build/configs/artik053/artik053_download.sh [hunk 1]
```diff
@@ -119,6 +119,26 @@ main()
...
+
+			# Download all binaries using openocd script
+			pushd ${OPENOCD_DIR_PATH}
+			${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
+			flash_write os ../../../../output/bin/tinyara_head.bin;	\
+			exit'
+			popd
+
+			# check romfs and download it
+			romfs
```
Comment (sunghan-chang) [inline build/configs/artik053/artik053_download.sh]: You wrote "OS Only", but here is romfs. What is right?

Comment (an4967) [reply build/configs/artik053/artik053_download.sh]: @sunghan-chang 'romfs' is a kind of user data, not binary. So i mean only os binary will be downloaded. If you want I will remove 'only' word.

Comment (sunghan-chang) [reply build/configs/artik053/artik053_download.sh]: OS and rom are different partitions.  When you say "OS only", including rom is wrong. And os and rom don't have any dependancy. We don't need to update both together.

**Comment by an4967** [comment]
@sunghan-chang I modified the echo comment and .flashSpec.xml

**Comment by sunghan-chang** [comment]
@an4967 Ok. I mis-communicated this with you. That's my fault. Anyway in partition map, there is "OS" and it specifies only os partition, not including romfs partition. So, with "os" option, downloading both, os and romfs is wrong. "OS+" or some option should be provided.

File: build/configs/artik053/artik053_download.sh [hunk 2]
```diff
@@ -119,6 +119,26 @@ main()
...
+		OS|os)
+			echo "OS :"
+
+			# check existence of os binary
+			if [ ! -f "${OUTPUT_BINARY_PATH}/tinyara_head.bin" ]; then
+				echo "TinyAra binary is not existed, build first"
+				finish_download 1
+			fi
+
+			# Download all binaries using openocd script
```
Comment (jeongchanKim) [inline build/configs/artik053/artik053_download.sh]: please modify the comment for description.
this is not for all binaries

**Comment by an4967** [comment]
@sunghan-chang Then, how about using "TIZENRT" as the option name? Because that option is for downloading tizenrt system including kernel & filesystem contents

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**